README.md

```diff
@@ -228,81 +229,99 @@ The following are the important variables you should configure when using this m
 
 To use this module, follow these steps:
 
-Copy and paste into your Terraform configuration, insert the variables, and run ``terraform init``:
+1. Copy and paste into your Terraform configuration, insert the variables, and run ``terraform init``:
 
-```hcl
-module "graphdb" {
-  source  = "Ontotext-AD/graphdb/aws"
-  version = "~> 1.0"
-
-  resource_name_prefix     = "graphdb"
-  aws_region               = "us-east-1"
-  ec2_instance_type        = "c5a.2xlarge"
-  graphdb_license_path     = "path-to-graphdb-license"
-  allowed_inbound_cidrs_lb = ["0.0.0.0/0"]
-}
-```
-Initialize the module and its required providers with:
+    ```hcl
+    module "graphdb" {
+      source  = "Ontotext-AD/graphdb/aws"
+      version = "~> 1.0"
 
-`terraform init`
+      resource_name_prefix     = "graphdb"
+      aws_region               = "us-east-1"
+      ec2_instance_type        = "m5.xlarge"
```

Comment:
Should match those from https://aws.amazon.com/marketplace/pp/prodview-kp53t5cg2jciq?sr=0-2&ref_=beagle&applicationId=AWSMPContessa